src/player.cpp

```diff
@@ -12942,6 +12976,23 @@ int player::get_hp_max( hp_part bp ) const
     return hp_total;
 }
 
+int player::get_stamina_max()
+{
+    return 1000;
```

Comment:
See previous discussion, I don't like tying it to strength, and it's out of scope for this PR IMO to add a new stat.
I want to get some feedback on how it plays before adding more complications like that.
The main thing holding it is that as it is now it's simply a huge buff to escaping from enemies as it makes the player strictly faster, and I'm experimenting with different things to compensate for that.